org.eclipse.gef/src/org/eclipse/gef/rulers/RulerProvider.java

```diff
@@ -142,8 +144,9 @@ public void getAccGuideValue(AccessibleControlEvent e, Object guide) {
 	 * @param guide the guide to which the model parts are attached.
 	 * @return list of attached model objects
 	 */
+	@SuppressWarnings("static-method")
 	public List getAttachedModelObjects(Object guide) {
-		return Collections.EMPTY_LIST;
```

Comment:
Is it intentional that this method still returns a raw list?

Reply:
At the time I was not sure about the type. After doing the getModelChildren() change in PR #501 I now know better. Changed it.